app/worker.py

```diff
@@ -136,8 +138,8 @@ async def build_response(self, reduced_results):
                     if library["Library Code"] == library_code:
                         response += library["Display name in Primo API"] + "\n"
                         break
-                
-                response += self.get_library_hours(library_code)
+
+                response += library_hours[library_code] + "\n"
```

Comment:
Might be better to use "get()" here just in case the library code is not there for some reason or wrap in a try except block